test/db/cmd/types

```diff
@@ -1334,9 +1334,9 @@ var char ** var_20h @ rbp-0x20
 var int64_t var_14h @ rbp-0x14
 var void * var_10h @ rbp-0x10
 var int64_t var_4h @ rbp-0x4
-arg int argc @ rdi
```

Comment:
I still think there is something wrong here. I would expect variables and arguments to be shown in the "right order", not just sorted by register names. I know this depends on aBI, etc. but it kinda made sense to have `rdi`, `rsi`, `rdx`, while I find this new sorting inappropriate in this particular context. @XVilka @thestr4ng3r @kazarmy WDYT?

Reply:
I agree, the argument order feels reversed. I'm not sure how to address that here though.

Reply:
this needs a little bit more investigation. Maybe it's just the way it is, but maybe we are missing something.